index.d.ts

```diff
@@ -1874,28 +1874,88 @@ export declare const Pill: BoxComponent<PillOwnProps, 'strong'>
 export type PopoverStatelessProps = BoxProps<'div'>
 
 export interface PopoverProps {
+  /**
+   * The position the Popover is on. Smart positioning might override this.
+   */
   position?: PositionTypes
+  /**
```

Comment:
This is just a personal ask, but could we be a little more explicit about this? I feel like it's a little hard for me to immediately understand at first, so something this would be helpful:

```ts
  /**
   * Controls whether the Popover is shown or not.
   * - When `true`, the component is manually shown.
   * - When `false`, the component is manually hidden.
   * - When `undefined`, the component is uncontrolled and the isShown state 
   *    is handled internally (i.e., retains default behavior and shows on hover)
   */
  isShown?: boolean
```

Reply:
Fair enough! How does it sound now?

Reply:
Much much better, thank you!!